DispatchGroup/ViewController.swift

```diff
@@ -59,7 +59,7 @@ class ViewController: UIViewController {
     
```

Comment:
dispatchQueue.async {
            self.getImageNetwork.getImage(imageName: "bmw") { [weak self] data in
                completion(data)
            }
        }

No need for [weak self]. Self is not used inside the closure